internal/bminventory/inventory.go

```diff
@@ -4594,6 +4660,10 @@ func (b *bareMetalInventory) updateInfraEnvData(ctx context.Context, infraEnv *c
 		updates["type"] = params.InfraEnvUpdateParams.ImageType
 	}
 
+	if params.InfraEnvUpdateParams.AdditionalTrustBundle != "" && params.InfraEnvUpdateParams.AdditionalTrustBundle != infraEnv.AdditionalTrustBundle {
```

Comment:
This would prevent us from unsetting the bundle - do we expect it to be just updatable once set?

Reply:
> This would prevent us from unsetting the bundle - do we expect it to be just updatable once set?

:thinking:

Reply:
This scenario:
* user sets the bundle
* changes it to some other one
* unsets it
would keep the bundle from step 2, right? I don't think we need `params.InfraEnvUpdateParams.AdditionalTrustBundle != ""` here - users are allowed to unset it

Reply:
oh yeah you're completely right, I'm just thinking :thinking: how to fix it. Because it's not so simple, the field needs to become nullable